### DISCUSSION TITLE
Coral USB Accelerator and SSD

### DISCUSSION THREAD

**Discussion by Kate222** [blakeblackshear/frigate#1284] [Ask A Question] [answered]
Does anyone use coral USB accelerator while using SSD to boot rpi? Is there any extra steps to get it work together? Thank you in advance.

**Comment by Ysbrand** [comment, marked as answer]
Yes, I've got a coral USB as well as a SSD connected to a PI4. I boot from the SSD as well.

Just make sure that you have enough power, in most cases you will need a powered USB3 hub to get enough power to the drive/coral

**Reply by stamandr** [reply]
Do you have any recommendations on a externally powered USB hub ?

**Reply by blakeblackshear** [reply]
There is a link in the docs: https://docs.frigate.video/installation#raspberry-pi-34
